get/all.go

```diff
@@ -134,6 +145,26 @@ func printItems(items []*unstructured.Unstructured, get Cmd, out io.Writer, head
 	return w.Flush()
 }
 
+func filteredListTypes(s *runtime.Scheme, kinds []string) ([]schema.GroupVersionKind, error) {
+	result := []schema.GroupVersionKind{}
+	lists := nineListTypes(s)
+	if len(kinds) == 0 {
+		return lists, nil
+	}
+OUTER:
+	for _, kind := range kinds {
+		for _, list := range lists {
+			if !strings.EqualFold(kind+"list", list.GroupKind().Kind) {
+				continue
+			}
+			result = append(result, list)
+			continue OUTER
```

Comment:
isn't it simpler to just `break` here?

Reply:
yeah, you are right 🤦

Reply:
It actually doesn't work as the error gets thrown when we found a list type for a requested kind now. So we can either leave it with the `label` or we introduce a `bool` var which allows to find out if something was found. Wdyt?

Reply:
Ah yeah makes sense. Both is fine with me honestly.

Reply:
I'll leave it then as it is. 👍